frontend/src/components/EntityList.js

```diff
@@ -10,29 +11,31 @@ class EntityList extends React.Component {
 
     constructor(props) {
       super(props);
-      this.state= { entities: []}
+      this.state= { objetosPerdidos: []}
     }
 
     componentWillMount() {
-      fetch(`${API_URL}/${this.props.entity}`)
+      fetch(`http://localhost:8888/objetosPerdidos`)
```

Comment:
Acá estaría bueno dejarlo con la variable, para cuando lo quieras subir. Sería así:

```suggestion
      fetch(`${API_URL}/objetosPerdidos`)
```